File: PRS Trade/PRSWord/CS/WordCore/Modules/About.cs
using System;
using DevExpress.Tutorials;
using System.Drawing;
using DevExpress.Utils.About;

namespace DevExpress.XtraRichEdit.Demos {
    public partial class About : ucAboutPage {
		public About() {
			InitializeComponent();
		}

        #region Properies
        protected override string ProductText { get { return "Microsoft Word® Style Rich Text Editor for WinForms.\r\n\r\nThe DevExpress Rich Text Editor allows you to introduce Microsoft Word like text editing capabilities into your next Windows Forms application with ease. The XtraRichEdit provides advanced text formatting capabilities and delivers numerous end-user options helping you deliver compelling business solutions for the Windows Forms platform."; } }
        protected override Image ProductImage { get { return DevExpress.Utils.ResourceImageHelper.CreateImageFromResources("DevExpress.XtraRichEdit.Demos.Images.XtraRichEdit.png", typeof(frmMain).Assembly); } }
        protected override Image ProductImageLight { get { return DevExpress.Utils.ResourceImageHelper.CreateImageFromResources("DevExpress.XtraRichEdit.Demos.Images.XtraRichEdit_light.png", typeof(frmMain).Assembly); } }
        protected override ProductKind ProductKind { get { return ProductKind.XtraRichEdit; } }
        #endregion
	}
}

File: PRS Trade/PRSCalend/VB/Scheduler/Modules/About.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports DevExpress.Tutorials
Imports DevExpress.Utils.About

Namespace DevExpress.XtraScheduler.Demos
	Partial Public Class About
		Inherits ucAboutPage
		Public Sub New()
			InitializeComponent()
		End Sub

		#Region "Properies"
		Protected Overrides ReadOnly Property ProductText() As String
			Get
				Return "Microsoft Office Style Scheduling Controls for WinForms" & Constants.vbCrLf & Constants.vbCrLf & "Features multiple calendar view types and side-by-side calendar display. Ships with complete Microsoft Outlook style end-user interface."
			End Get
		End Property
		Protected Overrides ReadOnly Property ProductImage() As Image
			Get
                Return DevExpress.Utils.ResourceImageHelper.CreateImageFromResources("XtraScheduler.png", GetType(frmMain).Assembly)
			End Get
		End Property
		Protected Overrides ReadOnly Property ProductImageLight() As Image
			Get
                Return DevExpress.Utils.ResourceImageHelper.CreateImageFromResources("XtraScheduler_light.png", GetType(frmMain).Assembly)
			End Get
		End Property
		Protected Overrides ReadOnly Property ProductKind() As ProductKind
			Get
				Return ProductKind.XtraScheduler
			End Get
		End Property
		#End Region
	End Class
End Namespace

